test/countBy.test.js

```diff
@@ -0,0 +1,57 @@
+import countBy from "../src/countBy.js";
+
+describe("countBy", () => {
+    it("can correctly count attributes from an array of objects", () => {
+        const users = [
+            { user: 'barney', active: true },
+            { user: 'betty', active: true },
+            { user: 'fred', active: false },
+        ];
```

Comment:
Users array is used in multiple places, so it could be defined once in the beginning
```suggestion
const users = [
  { user: 'barney', active: true },
  { user: 'betty', active: true },
  { user: 'fred', active: false },
];

describe("countBy", () => {
    it("can correctly count attributes from an array of objects", () => {
```